packages/patternfly-4/react-core/src/components/Nav/Nav.js

```diff
@@ -13,7 +13,7 @@ const propTypes = {
   /** Callback for when a list is expanded or collapsed */
   onToggle: PropTypes.func,
   /** Accessibility label */
-  'aria-label': PropTypes.string.isRequired,
+  'aria-label': PropTypes.string,
```

Comment:
looks like you removed the isRequired label her but did not add a defaultProp.

Reply:
Pushed a new commit with a default prop.